src/broker/cas_execute.c

```diff
@@ -916,6 +916,7 @@ ux_prepare (char *sql_stmt, int flag, char auto_commit_mode, T_NET_BUF * net_buf
       goto prepare_result_set;
     }
 
+  db_init_lexer_lineno ();
```

Comment:
why not direct call **csql_yyset_lineno**?

Reply:
csql_yyset_lineno() cannot be referenced in cas_execute.c without including parser.h, and 
I'm not sure including parser.h in cas_execute.c is structurally OK or not.

Reply:
db_open_buffer() is used in several places, including on line 883.
I think there is a need to review other areas other than the added 919 line.

Reply:
good point.